cli/flox-rust-sdk/src/models/manifest/typed.rs

```diff
@@ -988,9 +989,10 @@ impl SkipSerializing for Include {
 /// The structure for how a user is able to declare a dependency on an environment.
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
 #[cfg_attr(test, derive(proptest_derive::Arbitrary))]
+#[serde(deny_unknown_fields)]
 #[serde(
     untagged,
-    expecting = "Expected either a local or remote include descriptor."
+    expecting = "Must specify included environment as { dir = <dir>, [name = <name>] }"
```

Comment:
(non-blocking) I'm not sure how easy it will be to maintain this as we extend the descriptor schema.

Reply:
Didn't take long for it to go stale 🙈 https://github.com/flox/flox/pull/2972